core/processor/src/test/java/io/quarkus/annotation/processor/documentation/config/formatter/JavadocToAsciidocTransformerConfigItemTest.java

```diff
@@ -116,12 +116,12 @@ public void parseJavaDocWithStyles() {
 
     @Test
     public void parseJavaDocWithLiTagsInsideUlTag() {
-        String javaDoc = "List:" +
-                "<ul>\n" +
-                "<li>1</li>\n" +
-                "<li>2</li>\n" +
-                "</ul>" +
-                "";
+        String javaDoc = """
+                List:\
+                <ul>
+                <li>1</li>
+                <li>2</li>
+                </ul>""";
         String expectedOutput = "List:\n\n - 1\n - 2";
```

Comment:
using string blocks `"""` seems more readable imho and might be more performant as well considering several string concatenations (`"" + ""`) here.